apps/examples/testcase/le_tc/network/tc_net_pbuf.c

```diff
@@ -39,7 +39,7 @@
 * @brief		:check for the memory allocation for the pbuf by transport layer
 * @scenario		:
 * @apicovered		:pbuf_alloc()
-* @precondition 	:
+* @precondition		:
```

Comment:
why extra tab added

Reply:
because earlier, after * @precondition there was a space then tab, so after raising PR it gave me coding rules violations error.
so this is why i had to use extra tab but at vim editor these are aligned. not showing like here.